syntax = "proto3";

import "nanopb.proto";
package cybertx;

// option java_package = "fi.kapsi.koti.jpa.nanopb";
option java_package = "com.cybertx.protos";
// option java_outer_classname = "CyberTxProtos";

message UpdateLine {
    int32 line = 1;
    int32 channel = 2;
    int32 channel_value = 3;
}


message UpdateAll {
    int32 line = 1;
    repeated int32 channel_values = 2 [(nanopb).max_count = 16, (nanopb).fixed_length = true];;
}
